.env.example

```
@@ -0,0 +1,4 @@
+NODE_ENV=
+PORT=
+HOST=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HOST key should go before the NODE_ENV key

(UnorderedKey)